pkg/rtc/utils.go

```diff
@@ -126,7 +123,9 @@ func Recover(l logger.Logger) {
 		default:
 			err = errors.New("unknown panic")
 		}
-		l.Errorw("recovered panic", err, "panic", r)
+		l.Errorw("Panic", err, "panic", r)
+
+		os.Exit(1)
```

Comment:
I think this should not be here. It should be in `main`. Callers can choose to let things `recover` or exit. As a utility, this is just doing `recover`.